docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  proxy:
+    build:
+      context: .
+      dockerfile: ./development/docker/Dockerfile
+    command: bash -c "npm install && npm run proxy"
```

Comment:
Will it work on Win host? I have to use a list of commands close that you used in Vagrand provision.